Tranning/etc/frontend/di.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Framework\App\RouterList">
+        <arguments>
+            <argument name="routerList" xsi:type="array">
+                <item name="router" xsi:type="array">
```

Comment:
Naming convention - the "router" is a overly generic name here, something unit1CustomRoute might be better.